src/box/box.cc

```diff
@@ -6225,6 +6227,7 @@ box_free(void)
 	tuple_free();
 	port_free();
 	iproto_constants_free();
+	rt_access_destroy();
```

Comment:
From the [style guide](https://www.tarantool.io/en/doc/latest/contributing/c_style_guide/):

> For function naming we have a convention is to use:
>
> * new/delete for functions which allocate + initialize and destroy + deallocate an object,
> * create/destroy for functions which initialize/destroy an object but do not handle memory management,
> * init/free for functions which initialize/destroy libraries and subsystems.

Here we deinitialize the subsystem, so it should be named as `<...>_free()`.

Reply:
Renamed.